public/js/actions/sources.js

```diff
@@ -4,9 +4,9 @@
 "use strict";
 
 const defer = require("devtools/shared/defer");
-const { PROMISE } = require("devtools-sham/client/shared/redux/middleware/promise");
-const { Task } = require("devtools-sham/sham/task");
-const SourceUtils = require("devtools-sham/client/shared/source-utils");
+const { PROMISE } = require("../util/redux/middleware/promise");
+const { Task } = require("../util/task");
+const { isJavaScript } = require("../util/source");
```

Comment:
There are obviously other SourceUtils we will need soon. If SourceUtils is de-priveledged we can use that at somepoint.